render/detailed/connections.go

```diff
@@ -145,7 +145,7 @@ func outgoingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 				if !ok {
 					continue
 				}
-				key := newConnection(n, node, port, remoteEndpointID, localAddr)
```

Comment:
That's the whole point of this PR.

localEndpoint is the endpoint on the Internet node, corresponding to where an inbound connection from the Internet originated. It is the reverse-DNS of that endpoint we are interested in.

And we are using the remote port, which is the port *to* which the inbound internet connections are connected because we don't want to distinguish between connections from the same internet IP but different ports, but *do* want to distinguish between connections *to* different container/host IPs/ports.